aurora-mysql-plugin/src/test/java/io/cdap/plugin/aurora/mysql/AuroraMysqlPluginTestBase.java

```diff
@@ -115,9 +109,19 @@ public static void setupTest() throws Exception {
       BASE_PROPS.get(ConnectionConfig.PORT) + "/" + BASE_PROPS.get(ConnectionConfig.DATABASE);
     Connection conn = createConnection();
     createTestTables(conn);
+    tearDown = true;
     prepareTestData(conn);
   }
 
+  private static void getProperties() {
```

Comment:
why make this change? is it because the properties unavailable with the previous static initialization?

Reply:
Because the old code will fail if properties are unavailable. So I use `Assume.assume` to skip the test case if properties are not available.

While `Assume.assume` actually throws a specific runtime exception when assumption is false which is handled by outmost Junit Runner to skip the test case instead of failing the test case. Throwing runtime exception in static initialization will cause class initialization failure.